v4/feature/shared/src/main/java/exchange/dydx/trading/feature/shared/views/GradientSlider.kt

```diff
@@ -29,7 +29,7 @@ fun Preview_GradientSlider() {
 object GradientSlider {
     data class ViewState(
         val localizer: LocalizerProtocol,
-        val leftRatio: Float = -1.0f,
+        val leftRatio: Float = -0.5f,
```

Comment:
i'm confused, why did we change this?

Reply:
It's just for previewing..  actual value is provided by the view model.